examples/pubsub/publisher_test.cc

```diff
@@ -61,28 +61,28 @@ class PublisherServiceImpl : public tech::pubsub::PublisherService::Service {
  public:
   Status CreateTopic(::grpc::ServerContext* context,
                      const ::tech::pubsub::Topic* request,
-                     ::tech::pubsub::Topic* response) override {
+                     ::tech::pubsub::Topic* response) GRPC_OVERRIDE {
```

Comment:
I think GRPC_... we don't have a GPR in C++ land.

On Thu, Feb 26, 2015 at 2:10 PM Nicolas Noble <notifications@github.com>
wrote:

> In examples/pubsub/publisher_test.cc
> <https://github.com/grpc/grpc/pull/846#discussion_r25470001>:
>
> > @@ -61,28 +61,28 @@ class PublisherServiceImpl : public tech::pubsub::PublisherService::Service {
> >   public:
> >    Status CreateTopic(::grpc::ServerContext* context,
> >                       const ::tech::pubsub::Topic* request,
> > -                     ::tech::pubsub::Topic* response) override {
> > +                     ::tech::pubsub::Topic* response) GRPC_OVERRIDE {
>
> Same as above: GRPC_ or GPR_ ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/846/files#r25470001>.
>